rocksdb_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then 
```

Comment:
This only works for Linux distributions that use apt-get like ubuntu, maybe check if apt-get exists via something like:
if ! [ -x "$(command -v git)" ]; then

otherwise fallback to the Operating System not supported error